setup.py

```diff
@@ -12,18 +12,20 @@
 except ImportError:
     from distutils.core import setup
 
-dependencies = []
-
-html5lib_dependencies = [
-    'beautifulsoup4>=4.4.1,<5',
-    'html5lib>=0.999,<=1.0b10',
-]
-
-lxml_dependencies = [
-    'lxml>=3.6.0',
-]
+dependencies = {
+    # Keep this in sync with the dependencies in tox.ini.
+    'lxml': [
+        'lxml>=3.6.0,<5',
+    ],
+    'html5lib': [
+        'beautifulsoup4>=4.4.1,<5',
+        'html5lib>=0.999,<=1.0.1',
+    ],
+    'docs': [],
+}
```

Comment:
Not strictly necessary – just trying to make it less likely for `tox.ini` and this to go out of sync.